kong/cmd/utils/nginx_signals.lua

```diff
@@ -18,15 +18,15 @@ local function is_openresty(bin_path)
   local cmd = fmt("%s %s", bin_path, nginx_version_command)
   local ok, _, _, stderr = pl_utils.executeex(cmd)
   if ok and stderr then
-    log.debug("%s: '%s'", cmd, stderr:sub(1, -2))
     local version_match = stderr:match(nginx_version_pattern)
     if (not version_match) or (not nginx_compatible:matches(version_match)) then
-      return nil, "incompatible nginx found. Kong requires OpenResty, version "..tostring(nginx_compatible) ..
-        (version_match and ", got "..version_match or "")
+      log.verbose("incompatible nginx found at "..bin_path..". Kong requires OpenResty, version "..tostring(nginx_compatible) ..
+        (version_match and ", got version "..version_match or ""))
+      return nil
     end
     return true
   end
-  return nil, "could not determine nginx version in use. Kong requires OpenResty version "..tostring(nginx_compatible)
+  log.verbose("OpenResty 'nginx' executable not found at %s", bin_path)
```

Comment:
it includes the binary name, but remains a bit confusing; `OpenResty 'nginx' executable not found at nginx`.
I didn't include quotes because the other messages in the log didn't have quotes on paths either.